app/src/main/res/layout/row_item_description.xml

```diff
@@ -1,52 +1,83 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  android:layout_marginTop="12dp"
+  android:layout_marginBottom="@dimen/activity_margin_horizontal"
+  android:layout_marginStart="@dimen/activity_margin_horizontal"
+  android:layout_marginEnd="@dimen/activity_margin_horizontal"
+  android:layout_marginLeft="@dimen/activity_margin_horizontal"
+  android:layout_marginRight="@dimen/activity_margin_horizontal"
+  android:elevation="6dp">
+
+  <LinearLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:orientation="horizontal"
-    android:weightSum="8">
+    android:orientation="vertical">
 
-    <androidx.appcompat.widget.AppCompatSpinner
+    <RelativeLayout
```

Comment:
This layout would read a lot better if this was all a constraintLayout, I think given the scope of changes to this file we can do that